spec/activerecord-multi-tenant/multi_tenant_spec.rb

```diff
@@ -104,7 +104,7 @@ class SampleNestedTenant < ActiveRecord::Base
             end
             # rubocop:disable Layout/TrailingWhitespace
             # Trailing whitespace is intentionally left here
-            
```

Comment:
Not sure why trailing whitespace intentionally left here

Reply:
Editor autocorrect was running. It has been reverted 🙏